setup.py

```diff
@@ -103,6 +103,7 @@ def run(self):
 azure = ["adlfs==2021.8.2", "azure-identity>=1.4.0", "knack"]
 oss = ["ossfs==2021.8.0"]
 ssh = ["sshfs>=2021.8.1"]
+http = ["fsspec[http]>=2021.8.1", "aiohttp-retry==2.4.5"]
```

Comment:
DVC provides http support by default, so this should be part of `install_requires`.